Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
-    pagy (3.10.0)
+    pagy (3.11.0)
     parallel (1.20.1)
-    parser (3.0.0.0)
+    parser (3.0.1.0)
       ast (~> 2.4.1)
     pg (1.2.3)
-    popper_js (1.16.0)
+    popper_js (2.8.6)
```

Comment:
### [popper_js](https://github.com/glebm/popper_js-rubygem)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.16.0 → 2.8.6

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/glebm/popper_js-rubygem/compare/v1.16.0...v2.8.6).

These are the individual commits:
- (e372815) [Update to 2.6.0](https://github.com/glebm/popper_js-rubygem/commit/e3728152c0791c6439b055503b8cbc2eab6092f7)
- (d4b2eec) [bump dev rake version](https://github.com/glebm/popper_js-rubygem/commit/d4b2eec3a48b2b319637ec20cf02a5e0b29919f5)
- (f39bfc8) [.travis.yml: bump ruby](https://github.com/glebm/popper_js-rubygem/commit/f39bfc8e2b6c78f6343a7670a417f507c30ad2fa)
- (622d282) [Update to v2.8.6](https://github.com/glebm/popper_js-rubygem/commit/622d2826a963f48358c42aefdb38b29bf48522e6)


</details>